services-matching/src/main/java/com/mapbox/api/matching/v5/models/MapMatchingMatching.java

```diff
@@ -252,6 +262,15 @@ public DirectionsRoute toDirectionRoute() {
      */
     public abstract Builder voiceLanguage(@Nullable String voiceLanguage);
 
+    /**
+     * The universally unique identifier of the request that produced this route.
+     *
+     * @param requestUuid uuid
+     * @return this builder for chaining options together
+     */
+    @NonNull
```

Comment:
should be `@NonNull` added to every `Builder`'s methods?

Reply:
I don't really want to touch too many more classes in this PR unless I have to. We can follow up separately.